README_Lunchbot.md

```diff
@@ -1 +1,5 @@
 # lunchbot
+
+Below is the database model for Lunchbot displayed using mermaid.js. If you would like to modify the model, click on the image, make your changes through mermaid.live and then replace the markdown in this file with the new markdown.
+
+[![](https://mermaid.ink/img/pako:eNqtVF1v2jAU_SuWn0sZsNE2bxUNG9papASkPSBZJr5JLCU2cpxJLOS_zyEfjSFUqjQ_AD73cL_OvS5wIBlgB4N64TRSNN0JZM6r-7b10ek0GskCLZ437ve1t3J95KCYZgOUtffiegPmor5UJ9OKiwgJmsI7yIVGnNl3GYY8ANLCZf21Xi5XC5f82r4tfpAm3M3gQ-TiMip6BxjVYD6ONxJBy59WLlvf9dBJjkanNngv9tnYBWuqhpTy5AJTMuk60e9DW_BF2narqluegepnV6MpiPwaTXIRxEQqZv-libXauK9-W5H7e-M99ytqrL1mkxpy0B4SKaJewuQGmdReLZF6_P8oTq19nYKp5jyHV6PRmQcmZYB8KafmKfyV4iP5ekvT9PVcqYMCKTTlIrui3VB6YG-GZLanZlCsrqxGi1sitbJa8n8uu8BoFkl1JFdMntIISK66bdhLswdUIAYJaBjYAWt-PliHZrg1pMQ2WFBpteBzVQ05ag--wykos-bMvKZntzusYzAOsGN-Mghpnugd3onSUPNDNdUu41oq7IQ0yeAO01xL_ygC7GiVQ0tq3uWOlUhq6sROgfXxUD3dEc-0cWnmKuRRhZvmGjjW-pA543Flvo-4jvP9fSDTccZZTJWO_zzNx_Pp_JFOZzB_mNFvsxkL9pOnx3D6dRKyhy-TKcVlWf4DDzbO_w)](https://mermaid.live/edit#pako:eNqtVF1v2jAU_SuWn0sZsNE2bxUNG9papASkPSBZJr5JLCU2cpxJLOS_zyEfjSFUqjQ_AD73cL_OvS5wIBlgB4N64TRSNN0JZM6r-7b10ek0GskCLZ437ve1t3J95KCYZgOUtffiegPmor5UJ9OKiwgJmsI7yIVGnNl3GYY8ANLCZf21Xi5XC5f82r4tfpAm3M3gQ-TiMip6BxjVYD6ONxJBy59WLlvf9dBJjkanNngv9tnYBWuqhpTy5AJTMuk60e9DW_BF2narqluegepnV6MpiPwaTXIRxEQqZv-libXauK9-W5H7e-M99ytqrL1mkxpy0B4SKaJewuQGmdReLZF6_P8oTq19nYKp5jyHV6PRmQcmZYB8KafmKfyV4iP5ekvT9PVcqYMCKTTlIrui3VB6YG-GZLanZlCsrqxGi1sitbJa8n8uu8BoFkl1JFdMntIISK66bdhLswdUIAYJaBjYAWt-PliHZrg1pMQ2WFBpteBzVQ05ag--wykos-bMvKZntzusYzAOsGN-Mghpnugd3onSUPNDNdUu41oq7IQ0yeAO01xL_ygC7GiVQ0tq3uWOlUhq6sROgfXxUD3dEc-0cWnmKuRRhZvmGjjW-pA543Flvo-4jvP9fSDTccZZTJWO_zzNx_Pp_JFOZzB_mNFvsxkL9pOnx3D6dRKyhy-TKcVlWf4DDzbO_w)
```

Comment:
can we include just the text of the graph instead of a link to the `mermaid.ink`? github should render it for us

https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/